projects/opendr_ws/src/perception/scripts/object_detection_2d_ssd.py

```diff
@@ -22,11 +22,12 @@
 from opendr.engine.data import Image
 from opendr.perception.object_detection_2d import SingleShotDetectorLearner
 from opendr.perception.object_detection_2d import draw_bounding_boxes
+from opendr.perception.object_detection_2d.nms import Seq2SeqNMSLearner, SoftNMS, FastNMS, ClusterNMS
```

Comment:
```suggestion
from opendr.perception.object_detection_2d import Seq2SeqNMSLearner, SoftNMS, FastNMS, ClusterNMS
```

Reply:
the `init` file appears correct, hence it shouldn't require specifying this (as we don't do so for the other tools), but please double check it works

Reply:
As I am testing it, I have to modify the __init__.py on the object_detection_2d folder, so as to include all the imports of the _init__.py in object_detection_2d/nms folder. Then the _init__.py in object_detection_2d/nms will be a blank file. If that is more preferable, I will make those modifications today.

Reply:
Yes, personally I think it's preferable since that's how we do it for all other tools, so to have a consistent way of importing the learners is in my opinion a good thing @charsyme